docs/install_occlum_packages.md

```diff
@@ -0,0 +1,99 @@
+# Install Occlum Packages with Package Mangement Tool
+
+Occlum packages can be easily installed and deployed using mainstream package management tool including RPM and APT.
+
+
+### Prerequisite
```

Comment:
Why go straight from `# Heading level 1` to `### Heading level 3`, skipping `# Heading level 2`?